PrmPkg/PrmConfigDxe/PrmConfigDxe.c

```diff
@@ -157,6 +157,9 @@ SetRuntimeMemoryRangeAttributes (
                     (UINT64)RuntimeMmioRanges->Range[Index].Length,
                     // MU_CHANGE START: The memory space descriptor access attributes are not accurate. Don't pass
                     //                  in access attributes so SetMemorySpaceAttributes() doesn't update them.
+                    //                  EFI_MEMORY_RUNTIME is not a CPU arch attribute, so calling
+                    //                  SetMemorySpaceAttributes() with only it set will not clear existing page table
+                    //                  attributes for this region, such EFI_MEMORY_XP

```

Comment:
I assume this is meant to say `such as EFI_MEMORY_XP`.

Reply:
Thanks, fixed.